lib/CMakeLists.txt

```diff
@@ -352,6 +352,10 @@ if(QUDA_LAPLACE)
   target_compile_definitions(quda PUBLIC GPU_LAPLACE)
 endif(QUDA_LAPLACE)
 
+if(QUDA_TWOLINK_GSMEAR)
```

Comment:
is that used anywhere or can we just remove the `if` ?

Reply:
Sorry for the slow response. We intend for users to turn on this flag when building QUDA only if they need it. We will create a wiki page for it.

Reply:
I was more worried that there is no corresponding cmake option.

Reply:
Thanks for letting me know that! I have just added the cmake option.